admin-manual/installation-setup/customization/dashboard-config.rst

```diff
@@ -710,22 +711,69 @@ user, you can change it via the command-line:
 Security
 ^^^^^^^^
 
-Archivematica uses `PBKDF2 <http://en.wikipedia.org/wiki/PBKDF2>`_ as the default
-algorithm to store passwords. This should be sufficient for most users: it's
-quite secure, requiring massive amounts of computing time to break. However,
-other algorithms could be used as the following document explains:
-`How Django stores passwords <https://docs.djangoproject.com/en/1.4/topics/auth/#how-django-stores-passwords>`_ .
+Archivematica uses `PBKDF2`_ as the default algorithm to store passwords. This
+should be sufficient for most users: it's quite secure, requiring massive
+amounts of computing time to break. However, other algorithms could be used as
+the following document explains: `How Django stores passwords`_ .
 
 Our plan is to extend this functionality in the future adding groups and
 granular permissions support.
 
-.. _admin-handle-server:
+.. _admin-handlenet:
 
 Handle server config
 --------------------
-Archivematica can to be configured to make requests to a Handle System HTTP API
-so that files, directories and entire AIPs can be assigned persistent
-identifiers (PIDS) and derived persistent URLs (PURLs).
+
+Archivematica can mint persistent identifiers (PIDs) for digital objects,
+directories, or AIPs by defining the PIDs in a configured `Handle.Net`_
+registry. Handle.Net can then create persistent URLs (PURLs) from the PIDs and
+can reroute requests to the persistent URLs to a target URL that is configured
+in Handle.Net.
+
+.. image:: images/handlenet-config.*
+   :align: right
+   :width: 45%
+   :alt: Handle.Net configuration settings
+
+Fields:
+
+* **Web service endpoint**: The URL for (POST) requests to create and resolve
+  PIDs.
+* **Web service key**: Web service key needed for authentication to make
+  PID-binding requests to the PID web service endpoint.
+* **Naming authority**: Handle naming authority (e.g., 12345)
+* **Resolver URL**: The URL to append generated PIDs to in order to create
+  (potentially qualified) PURLs (persistent URLs) that resolve to the applicable
+  resolve URL. Note the second "r" in "resolver"!
+* **AIP PID source**: The source of the AIP's persistent identifier. The UUID of
+  the AIP is the default since it is virtually guaranteed to be unique. However,
+  the accession number of the transfer may be used, assuming the user can
+  guarantee a 1-to-1 relationship between the transfer and the AIP.
+* **Verify SSL certificates**: Selecting this box will ensure that Archivematica
+  verifies SSL certificates when making requests to bind PIDs.
+* **Archive resolve URL template**: Template (Django or Jinja2) for the URL that
+  a unit's PURL should resolve to. Has access to "pid" and "naming_authority"
```

Comment:
Maybe for these "Has access to" variable names you should use code formatting instead of double quotes.